collections/_article/dev-snapshot-godot-4-1-beta-1.md

```diff
@@ -0,0 +1,120 @@
+
+  - Add an option to use model space with the `look_at` method and other similar methods ([GH-76082](https://github.com/godotengine/godot/pull/76082)).
+  - Switch "front" and "back" camera views in the editor to be consistent with itself ([GH-76052](https://github.com/godotengine/godot/pull/76052)).
+  - Fix a long-standing `PathFollow` issue with the forward direction ([GH-72842](https://github.com/godotengine/godot/pull/72842)).
+
+Finally, Godot 4.1 brings back frame delta smoothing, first introduced in Godot 3.4 ([GH-52314](https://github.com/godotengine/godot/pull/52314)) by [lawnjelly](https://github.com/lawnjelly). You can read more about it in our [3.4 release blog post](/article/godot-3-4-is-released).
+
+### C#
+
+The team of .NET contributors have mostly bridged the gap between C# and GDScript in Godot 4 in terms of feature parity. However, one glaring missing feature was global classes. In GDScript you can register any class as globally accessible with the `class_name` keyword, which, among other things, makes it appear in various parts of the editor UI. For example, you can create custom resources and instantiate them from the appropriately typed fields in the inspector dock. C# has lacked this functionality, but thanks to efforts from [Raul Santos](https://github.com/raulsntos) and [Will Nations](https://github.com/willnationsdev) (as well as rigorous testing from many other contributors) this is no longer the case ([GH-72619](https://github.com/godotengine/godot/pull/72619)). You can get started with this new feature by reading the [freshly updated documentation](https://docs.godotengine.org/en/latest/tutorials/scripting/c_sharp/c_sharp_global_classes.html) on global classes in Godot C#.
+
+Another area of C# development that received attention during this release cycle is configurability. For example, [Alex de la Mare](https://github.com/alexdlm) provided an option for fine-grained disabling of source generators ([GH-71049](https://github.com/godotengine/godot/pull/71049)) and [RedworkDE](https://github.com/RedworkDE) made include scripts customizable on the export preset level instead of a global setting ([GH-72896](https://github.com/godotengine/godot/pull/72896)). This later features comes with a bigger improvement beyond just .NET, as all editor export plugins can now define custom export options for users to provide ([GH-72895](https://github.com/godotengine/godot/pull/72895)).
```

Comment:
I'm not sure I understand "This later features comes with...", is this grammatically correct? Was it meant to be "The latter features come with..." (i.e. "The features we last mentioned come with...").

Might need rephrasing either way to be easier to understand.

Reply:
Should be "latter feature", of course!

Reply:
"The latter feature comes with" might be fine already. I mostly felt the need to rephrase if this had to be plural and refer the the last X mentioned features which would be a bit of an unconventional construct.

Reply:
```suggestion
Another area of C# development that received attention during this release cycle is configurability. For example, [Alex de la Mare](https://github.com/alexdlm) provided an option for fine-grained disabling of source generators ([GH-71049](https://github.com/godotengine/godot/pull/71049)) and [RedworkDE](https://github.com/RedworkDE) made include scripts customizable on the export preset level ([GH-72896](https://github.com/godotengine/godot/pull/72896)), where previously you would have a global project setting. In fact, this last feature comes with an positive change for anyone creating editor export plugins, as they now have a new API to define custom export options for users to supply ([GH-72895](https://github.com/godotengine/godot/pull/72895)).
```